jupyterhub/templates/scheduling/_scheduling-helpers.tpl

```diff
@@ -14,6 +14,17 @@
 {{- $dummy := set . "tolerationsList" (include "jupyterhub.tolerationsList" .) }}
 {{- $dummy := set . "tolerations" (include "jupyterhub.tolerations" .) }}
 {{- $dummy := set . "hasTolerations" .tolerations }}
+
+{{- $dummy := set . "nodeAffinityRequired" (include "jupyterhub.nodeAffinityRequired" .) }}
+{{- $dummy := set . "podAffinityRequired" (include "jupyterhub.podAffinityRequired" .) }}
+{{- $dummy := set . "podAntiAffinityRequired" (include "jupyterhub.podAntiAffinityRequired" .) }}
+{{- $dummy := set . "nodeAffinityPreferred" (include "jupyterhub.nodeAffinityPreferred" .) }}
+{{- $dummy := set . "podAffinityPreferred" (include "jupyterhub.podAffinityPreferred" .) }}
+{{- $dummy := set . "podAntiAffinityPreferred" (include "jupyterhub.podAntiAffinityPreferred" .) }}
+{{- $dummy := set . "hasNodeAffinity" (or .nodeAffinityRequired .nodeAffinityPreferred) }}
+{{- $dummy := set . "hasPodAffinity" (or .podAffinityRequired .podAffinityPreferred) }}
+{{- $dummy := set . "hasPodAntiAffinity" (or .podAntiAffinityRequired .podAntiAffinityPreferred) }}
+{{- $dummy := set . "hasAffinity" (or .hasNodeAffinity .hasPodAffinity .hasPodAntiAffinity) }}
```

Comment:
I need a lot of helpers, some that contains the output at a level where I can augment it with other default output from affinity presets in future PRs.